source/Halibut.Tests/TrustFixture.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Halibut.Tests.Support;
+using Halibut.Tests.Support.TestAttributes;
+using Halibut.Tests.Support.TestCases;
+using Halibut.Tests.TestServices.Async;
+using Halibut.TestUtils.Contracts;
+using NUnit.Framework;
+
+namespace Halibut.Tests
+{
+    public class TrustFixture : BaseTest
+    {
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testListening: true, testPolling: false, testWebSocket: false)]
```

Comment:
Should polling and websockets be tested as well?

Reply:
The polling and websocket tests didn't exercise the code as they didn't start listeners. The `TrustOnly` call only clears listeners.

Would this effectively be testing polling/websockets anyway because Halibut is always listening on one side of the connection? In listening mode, Tentacle is the listener, and in polling/websocket, Octopus Server is the listener. Either way, the Halibut listening code is being exercised.